README.md

```diff
@@ -40,7 +40,14 @@ sudo snap install --edge nvim --classic
 
 [Other methods of installation are available in the official repository of Neovim](https://github.com/neovim/neovim/wiki/Installing-Neovim)
 
-Instructions for **NodeJS** are also available in [the official repo.](https://github.com/nodesource/distributions/blob/master/README.md)
+
+To install **NodeJS** in the most convinient way, install NodeJS via NVM (Node Version Manager).
+```
+curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.38.0/install.sh | bash
+# restart terminal
```

Comment:
Why  restart of terminal is required?
Give the official source to the documentation for this method

Reply:
Sometimes there can be [some troubleshoots](https://github.com/nvm-sh/nvm#troubleshooting-on-linux), therefore reopening terminal is required.

I'll add this link into comment.